linux_os/guide/system/network/network-iptables/package_iptables-services_installed/rule.yml

```
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Install iptables-services Package'
+
+description: |-
+    {{{ describe_package_install(package="iptables-services") }}}
+
+rationale: |-
+    <tt>iptables-services</tt> permits to load iptables rules during the system startup or
+    to reload iptables rules in runtime.
```

How including some of the text from the RPM description?

```
$ rpm -qi iptables-services
...
URL         : http://www.netfilter.org/projects/iptables
Summary     : iptables and ip6tables services for iptables
Description :
iptables services for IPv4 and IPv6

This package provides the services iptables and ip6tables that have been split
out of the base package since they are not active by default anymore.
```


```suggestion
    <tt>iptables-services</tt> provides the services iptables and ip6tables that have been split
out of the base package since they are not active by default anymore.
    These services load the iptables rules during the system startup and also allow one to reload the iptables rules during runtime.
```

---

Good idea. Let me update it.